packages/internals/src/utils/canPrompt.ts

```diff
@@ -1,4 +1,5 @@
 import prompt from 'prompts'
+
```

Comment:
This change is unexpected, looks like a merge went wrong

Reply:
This one is the linting doing this.